src/github/credentials.ts

```diff
@@ -25,6 +28,15 @@ export class CredentialStore {
 		this._configuration = configuration;
 		this._octokits = new Map<string, Octokit>();
 		this._authenticationStatusBarItems = new Map<string, vscode.StatusBarItem>();
+		vscode.commands.registerCommand(AUTH_INPUT_TOKEN_CMD, async () => {
+			const uriStr = await vscode.window.showInputBox({ prompt: 'Token' });
+			if (!uriStr) { return; }
+			const uri = vscode.Uri.parse(uriStr);
```

Comment:
I think we should try-catch this

Reply:
What would we do with it? Currently, if I paste in a malformed URI (rather hard to do, really) the error bubbles up to the UI:

<img width="474" alt="screen shot 2018-10-19 at 1 00 34 pm" src="https://user-images.githubusercontent.com/7763406/47232908-13306680-d39f-11e8-9ac6-75ea55b5e7bf.png">

Which seems right?